routing/payment_lifecycle.go

```diff
@@ -247,7 +247,18 @@ func (p *paymentLifecycle) resumePayment() ([32]byte, *route.Route, error) {
 
 		// We found a route to try, launch a new shard.
 		attempt, outcome, err := shardHandler.launchShard(rt)
-		if err != nil {
+		switch {
```

Comment:
Not sure why I cannot comment on it, but did we resolve this convo? https://github.com/lightningnetwork/lnd/pull/5214#discussion_r614453436

Reply:
Ah yeah sorry, thought I'd replied! We will indeed hit `state.terminate || state.remainingAmt == 0` and then wait for our shards 👍